spring-kafka/src/main/java/org/springframework/kafka/listener/KafkaMessageListenerContainer.java

```diff
@@ -1462,6 +1463,16 @@ else if (ackMode.equals(AckMode.COUNT_TIME) && elapsed) {
 		}
 
 		private void processSeeks() {
+			List<TopicPartitionOffset> timestampSeeks = this.seeks.stream()
+				.filter(tpo -> SeekPosition.TIMESTAMP.equals(tpo.getPosition()))
+				.collect(Collectors.toList());
```

Comment:
I understand that this is not critical path in our logic, but such a `processSeeks()` method is called on every poll.
So, can we consider do not use Java Streams? According your logic it looks like a simple `Iterator` can do the same.